Clean-up/Terraform/main.tf

```diff
@@ -0,0 +1,79 @@
+# This file terraforms lambda and eventbridge on AWS
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.AWS_REGION
```

Comment:
Do the access key and secret key not also go here?

Reply:
Added